docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[searchable-snapshots]]
+== {search-snaps-cap}
+
+{search-snaps-cap} enable you to significantly reduce costs by 
+leveraging external storage for read-only data. 
+Searchable snapshots can be used in two ways:
+
+* Snapshot-backed indices: Instead of maintaining replicas of your data within the cluster, 
+a snapshot-backed index relies on a searchable snapshot for redundancy. 
+You can perform all regular data retrieval operations, but cannot write to the index. 
+In the event of a failure, data is recovered from the snapshot. 
+Latency increases during recovery, but you can continue to query your data.
```

Comment:
Maybe add that searchable snapshots are eagerly downloaded and once fully downloaded, search performance against searchable snapshots will be similar to search performance against a regular index.